### PR TITLE
Sphinx-compatible docs

### LINKED ISSUE #42: Improve documentation for Stonesense
The Stonesense readme isn't particularly comprehensive, and parts are certainly out of date.  It would be good to write more comprehensive docs, like the new DFHack documentation.  I'll try to get to this sometime, but probably not before a couple of DF releases.  If done appropriately, it could be integrated in the same way as the df-structures docs on https://dfhack.readthedocs.org

Related to DFHack/dfhack#648

### PR DESCRIPTION
**Pull request by PeridexisErrant**
You could probably convert the wiki pages, notes in init files, text
explanations of how sprites work, etc into a set of pages, but then who
would maintain them?  I think this is a nice compromise.

Closes #42

### CODE REVIEW

**Comment by lethosor** [comment]
Any hooks need to be set up for this?

**Comment by PeridexisErrant** [comment]
If it's otherwise ready to merge, how about I reconcile the link targets here and in the DFHack repo?

That would avoid the dual problem of an unlinked document and duplicate information in the Plugins docs; I'll add corresponding commits here and in the DFHack repo later today unless I hear otherwise.

File: docs/Stonesense.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+:orphan:
+
+.. stonesense:
```
Comment (PeridexisErrant) [inline docs/Stonesense.rst]: @lethosor this is very embarrassing, but I made a typo here.  In order to actually make the links (and pass the build), this line must be `.. _stonesense:` - it's missing the underscore. I'm so sorry!

Comment (lethosor) [inline docs/Stonesense.rst]: Fixed in bd6e8f9, thanks

**Pull request merged**